src/index.tsx

```diff
@@ -1181,7 +1182,6 @@ class App extends React.Component<{}, AppState> {
             let resizeHandle: string | false = false;
             let isDraggingElements = false;
             let isResizingElements = false;
-            const cursorStyle = document.documentElement.style.cursor;
```

Comment:
I'm not sure what state this was preserving? It seemed to always be `''` on my machine.

Reply:
I didn't write this but I personally have the habit of restoring things to whatever they were before.